package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-braintree-xplat",
-  "version": "2.2.1",
+  "version": "2.3.0",
```

Comment:
The nonce will be verified when first used on the server, and not immediately. this is fine, but technically not a strictly bugfix, as there's a slight variation in behavior. Most users will never know